docs/api/asyncio_con.rst

```diff
@@ -550,3 +591,161 @@ Connection Pools
     .. py:attribute:: free_size
 
         Number of available connections in the pool.
+
+
+.. _edgedb-python-asyncio-api-transaction:
+
+Transactions
+============
+
+The most robust way to execute transactional code is to use
+the ``retry()`` loop API:
```

Comment:
```suggestion
The most robust way to execute transactional code is to use
the ``retry()`` API:
```

Reply:
Actually, never mind, let's keep "loop API". I think it better communicates to the reader that hey, this weird looking construct is actually the api.